File: Leap year Calculator/LeapYearCalculator.swift
func isLeapYear(year:Int)->Bool{
    if((year>=1 && year < 9999) && ((year%4==0 && year%100>0)||(year%400==0))){
        return true;
    }
    return false;
}
print(isLeapYear(year: 2017))

File: Leap year Calculator/LeapYearCalculator.php
<?php
function isLeapYear($year){
    if(($year>=1 && $year <=9999) && (($year%4==0 && $year%100>0)||($year%400==0))){
        echo "True";
    }else{
        echo "False";
    }
}

isLeapYear(4000);
